src/DynamoCore/Properties/Resources.en-US.resx

```diff
@@ -810,4 +810,19 @@ This package has not been loaded because another conflicting package was loaded
   <data name="NodeUnhandledMsg" xml:space="preserve">
     <value>Unhandled 'DummyNode.NodeNature' value: {0}</value>
   </data>
+  <data name="HideWiresPopupMenuItem" xml:space="preserve">
+    <value>Hide Wires</value>
+  </data>
+  <data name="UnhideWiresPopupMenuItem" xml:space="preserve">
+    <value>Unhide Wires</value>
+  </data>
+  <data name="PackageManagerPackageHasNoDependencies" xml:space="preserve">
+    <value>No dependencies.</value>
+  </data>
+  <data name="PackageManagerPackageUpdated" xml:space="preserve">
+    <value>Updated</value>
+  </data>
+  <data name="PackageManagerPackageNew" xml:space="preserve">
+    <value>New</value>
+  </data>
```

Comment:
Can you indicate where are these two used?

Reply:
On the bubble next to the package in the results view. Appears if it's New or recently Updated

Reply:
Then I would also move them to `DynamoCoreWpf` because they are utilized there but not in DynamoCore project?

Reply:
Moved